block-producer/src/state_view/mod.rs

```diff
@@ -131,6 +134,7 @@ where
 /// 1. the candidate transaction doesn't modify the same account as an existing in-flight transaction
 /// 2. no consumed note's nullifier in candidate tx's consumed notes is already contained
 /// in `already_consumed_nullifiers`
+#[instrument(ret, err(Debug), fields(COMPONENT))]
```

Comment:
I don't think we need to log arguments here and probably can skip logging `err` and `COMPONENT` (assuming they are already captured by the parent spans.

Also, maybe this should be instrumented at `debug` level?